braid/_braid.h

```diff
@@ -278,6 +280,9 @@ typedef struct _braid_Core_struct
    braid_Int     done;             /**< boolean describing whether XBraid has finished */
    braid_Int     wrapper_test;     /**< boolean describing whether this call is only a wrapper test */
    braid_Int     calling_function; /**< from which function are we accessing the vector */
```

Comment:
Maybe call this tloc_upper and tloc_lower, or something similar (?) to denote that these are local time values?

Reply:
Changed in addb4ab5ec2fbb34481dd5f5190a9740ca0aee22